2017-11-13-snare.markdown

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title:  "Snare"
+date:   2017-11-13
+source: Xanathar.165
+tags: [ druid, ranger, wizard, level1, abjuration]
+---
+
+**1st-level Abjuration**
+
+**Casting Time**: 1 minute
+
+**Range**: Touch
+
+**Components**: S, M (25 feet of rope, which the spell consumes)
+
+**Duration**: 8 hours
+
+  As you cast this spell, you use the rope to create a circle with a 5-foot radius on the ground or the floor. When you finish casting, the rope disappears and the circle becomes a magic trap.
```

Comment:
Do not indent new paragraphs. Instead leave a blank line between paragraphs. That's the style we've been using for the rest of the posts.